packages/fast-permutator/__tests__/test-utils/index.js

```diff
@@ -0,0 +1,283 @@
+var get = require('lodash').get,
```

Comment:
are we avoiding const here for a reason? I believe that the issue Aaron was running into was due to the fact that TravisCI was running in node v 0.10 - he's update to use ^4.0 and const doesn't seem to be an issue anymore.